reward_score/readme_gen_pred.py

```
@@ -0,0 +1,111 @@
+import difflib
+from utils import register_reward_score, get_llm_response
+from concurrent.futures import ThreadPoolExecutor, TimeoutError
+
+# 单例线程池（可以在模块全局初始化一次）
```

Ruff (0.12.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)